### ISSUE TITLE
RubyMotion project

### ISSUE THREAD

**Issue by FLemon** [FLemon/FLemon.github.io#1]
# RubyMotion Project
## tl;dr

RubyMotion is great, development in ruby, supports production of IOS app and ANDROID app.
I want to take its advantages to develop an app using RubyMotion, for both app platforms in a single project directory. 

Because they are in ruby, there must be common parts between both platforms to extract in to a shared component. Achieving this may means a lot less development time for releasing an app for both platforms, and less learning curve, no more objective c(or swift) and java, but only ruby, yes only ruby...

check it out: https://github.com/FLemon/motion-playground
P.S. the android version may not be in sync with the IOS, because I develop IOS version first
